pkg/ocm/ocmWrapper.go

```diff
@@ -45,9 +46,26 @@ type DefaultOCMInterfaceImpl struct{}
 
 var DefaultOCMInterface OCMInterface = &DefaultOCMInterfaceImpl{}
 
+func (o *DefaultOCMInterfaceImpl) SetupOCMConnection() (*ocmsdk.Connection, error) {
+	ocmNotLoggedInMessage := "Not logged in"
+
+	// Setup connection at the first try
+	connection, err := ocm.NewConnection().Build()
```

Comment:
Can we cache the connection in `DefaultOCMInterfaceImpl` struct for optimized performance?

Reply:
As currently, in each function, it will close the connection in the end. So the connection itself it is reusable.
Will create a new card to handle this cached connection case.